clients/fides-js/src/components/tcf/TcfVendors.tsx

```diff
@@ -158,26 +119,28 @@ const DataCategories = ({
   );
 };
 
-const StorageDisclosure = ({ vendor }: { vendor: Vendor }) => {
+const StorageDisclosure = ({ vendor }: { vendor: VendorRecord }) => {
   const {
     name,
-    usesCookies,
-    usesNonCookieAccess,
-    cookieMaxAgeSeconds,
-    cookieRefresh,
+    uses_cookies: usesCookies,
+    uses_non_cookie_access: usesNonCookieAccess,
+    cookie_max_age_seconds: cookieMaxAgeSeconds,
+    cookie_refresh: cookieRefresh,
   } = vendor;
   let disclosure = "";
   if (usesCookies) {
```

Comment:
Do we need to change this to `uses_cookies` or similar, the conditional isn't working as I'd expect.

This value is false, but cookie refresh is true, and I see this:
![Screenshot 2023-10-18 at 5 02 04 PM](https://github.com/ethyca/fides/assets/9755598/dcd70aa8-9067-4ca6-a42d-54f254d8f4e6)
e,

Reply:
I wasn't able to reproduce, but while looking into this I did notice that there was a gap if there are no cookies + no privacy/legint URLs, so I fixed that 
![image](https://github.com/ethyca/fides/assets/24641006/419a2b44-dc99-4e7e-a071-05707b042737)

Reply:
ah sorry false alarm.  Seeing better handling needed when retention period is null?
![Screenshot 2023-10-19 at 12 45 08 PM](https://github.com/ethyca/fides/assets/9755598/12da7411-2a26-4862-b696-35e333de2e5b)

Reply:
ah, good catch!

Reply:
Allison what I think I was doing yesterday was not always hitting save after those toggles 😫.  I need to spend more time in the UI.

Reply:
okay, fixed! https://github.com/ethyca/fides/pull/4286/commits/c49d5e5c172991a04d66b3c5acd706cdd69bf52d

![image](https://github.com/ethyca/fides/assets/24641006/111bd622-5b1e-4dde-bfd2-7a9f2aca992f)